### PR TITLE
Update FAQ with images and text changes

### LINKED ISSUE #111: FAQ Edits
Suggested tweaks or changes to the FAQ:

**What is IoT?**
I feel like we need a bit more here to give it an understanding that it's not just devices but that it's also everyday objects that now have increased functionality as a result of IoT. Google's definition gets us a bit closer I think:
_The interconnection via the Internet of computing devices embedded in everyday objects, enabling them to send and receive data._

**Why would I want to use this?**
Again I think this is a bit too general. I think we might want to give a bit more info or context. While the blurb below is a bit wordy I think it gets to the idea a bit more powerfully.
_The Internet of Things allows us to change our relationship with the objects in our everyday lives. While there is an increase in "smart" objects (like fridges, home heating, and even lighting), most of the time we can only consume what others give us. With App Inventor + IoT you can have the power to build things yourself - decide exactly how you want to interact with the world around you, and have more control over the data you share_

**How can I support MIT App Inventor?**
I would add a link the word "forums" to the forums

**How do I import extensions?**
I really think we need a picture here - or a link to something that shows this. It's too abstract with just words

**I didn't find help for my issue. What should I do?**
I would capitalize "Tutorial" and provide links to that and the How Tos 

**Why don't I see my device after scanning for Bluetooth?**
Do we want to show pictures of the reset buttons?

**I built a MIT App Inventor app to connect my IoT board, but my Android device cannot find my board while scanning. What should I do?**
I think you might want to put the terms "backed off" and "power cycling" in layman's terms

**Uploading the sketch is failing/My Arduino won't connect, what should I do?**
I would bold the following headers:
I get the message "Error compiling board for Arduino/Genuino Uno" or similar.
I get the message "An error occurred while uploading the sketch"
I get the message "Your Arduino firmware is out of date" or "Your Arduino firmware does not match the expected version"

**How do I check the firmware for my Arduino 101?**
A screenshot wold be good here

**How do I rename my Arduino?**
Provide the file name

**When I scan for Bluetooth devices I do not see my Micro:bit. Help!**
I would add "See below for how to pair your Micro:bit"

### PR DESCRIPTION
**Pull request by ewpatton**
Fixes #111

### CODE REVIEW

**Comment by ewpatton** [comment]
Also tagging @hnichols9560 for grammar review.

**Comment by hnichols9560** [comment]
Thanks Evan!  Here are a few more changes.

Hedge

[ ]
By networking devices in our everyday lives, we can leverage those devices as a means of interacting with the world by sending and receiving data, and by sensing and effecting the environment in which things reside.	
By networking devices in our everyday lives, we can interact with the world by sending and receiving data, and by sensing and effecting the environment in which things reside.

[ ]
While there is an increase in "smart" objects (like refridgerators, home heating, and even lighting), most of the time we are only passive users of technology.	
While there is an increase in "smart" objects (like refrigerators, home heating, and even lights), most of the time we are only passive users of technology.

[ ]
There is also a free book) available online for your use in learning MIT App Inventor.	
There is also a free book available online for your use in learning MIT App Inventor.

[ ]
You will need an Android phone or tablet with harrdware for Bluetooth 4.0 or higher, running Android 5.0 (Lollipop) or higher to access the Bluetooth hardware.	
You will need an Android phone or tablet with hardware for Bluetooth 4.0 or higher, running Android 5.0 (Lollipop) or higher to access the Bluetooth hardware.

[ ]
This allows extensions to be published by developers other than MIT and to be included or upgraded in projects at your discretion.	
This allows extensions published by developers other than MIT to be included or upgraded in projects at your discretion.

[ ]
If you have other apps running the background on your mobile device that have established a connection to your Bluetooth low energy device, that app may maintain its connection in the background.	
If you have another app running in the background on your mobile device with a connection to your Bluetooth low energy device, that app may still be connected.

[ ]
You may need to stop and restart scanning as some Bluetooth chipsets may slowly increase the time between scans to reduce power consumption.	
You may need to stop and restart scanning, as some Bluetooth chipsets may slowly increase the time between scans to reduce power consumption.

[ ]
Changing the name in quotes from AppInventor to another name and then uploading the changes to the Arduino will cause the Arduino to advertise under the new name.	
Change the name in quotes from AppInventor to another name and then upload the change to the Arduino to use the new name.

[ ]
I didn't find help for my issue. What should I do?	
This is midway through the first section.  I think it should be the last question in the FAQ.
